### DISCUSSION TITLE
Filter by null does not work when using the relationId field

### DISCUSSION THREAD

**Discussion by masterbater** [prisma/prisma#20014] [Q&A]
Works
```typescript
"AND": [
    {
      "client": null
    }
  ]
```
Doesnt work
```typescript
"AND": [
    {
      "clientId": null
    }
  ]
```

Schema

```prisma
 clientId               String?                      @map("client") @db.ObjectId
    client                 clients?                     @relation(fields: [clientId], references: [id])
```

**Comment by nurul3101** [comment]
Hello @masterbater 👋 

Can you please share the query that you are invoking?
Which version of Prisma are you using?
Sharing the model would also help in replicating this locally.

**Reply by nurul3101** [reply]
@masterbater I am checking to see if you had a chance to look at my last response.

It would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 
Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Reply by nurul3101** [reply]
We are closing this Discussion due to inactivity. Please open a new Discussion if you have a similar problem 🙏
